bower.json

```diff
@@ -8,7 +8,7 @@
     "ember-qunit": "0.3.1",
     "ember-qunit-notifications": "0.0.7",
     "ember-resolver": "~0.1.15",
-    "jquery": "^1.11.1",
+    "jquery": "1.11.1",
```

Comment:
Have to update ember (1.13.12 / 2.2.2. / > 2.3.0) or stick to jquery < 2.x due to this bug http://emberjs.com/blog/2016/01/15/ember-2-3-released.html#toc_removing-the-jquery-version-assertion. But I don't want to address this in my PR. Just committed the change necessary for running tests by accident.